src/micro_http/src/request.rs

```diff
@@ -511,5 +511,15 @@ mod tests {
         assert_eq!(request.headers.expect(), false);
         assert_eq!(request.headers.content_length(), 0);
         assert!(request.body.is_none());
+
+        let request = Request::try_from(b"GET http://localhost/ HTTP/1.0\r\n\
+                                                                                    Accept-Encoding: identity;q=0\r\n\r\n");
```

Comment:
Consider using `cargo fmt --all` to automatically format your code. I think this indentation would also go away but if not, please bring it back a bit.